requirements-optional.txt

```diff
@@ -0,0 +1,31 @@
+# Optional dependencies of Terra that can (mostly) reliably be installed with
+# `pip`.  This file is still called `requirements-optionals.txt` just to match
+# standard pip conventions, even though none of these are required.
+#
+# If updating this, you probably want to update `qiskit.utils.optionals` too.
+
+qiskit-aer
+qiskit-toqm>=0.0.4; platform_machine != 'aarch64' or platform_system != 'Linux'
```

Comment:
Do we need toqm anymore after #9042 and #9057?

Reply:
Oh, after #9057 there's no point any more.  Removed in 535f270f9.